health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/Map.js

```
@@ -0,0 +1,369 @@
+          display: "flex",
+          flexDirection: "column",
+          zoom: pageZoom ? 1 : 1.25,
+        }}
+        onMouseMove={(e) => {
+          const rect = e.currentTarget.getBoundingClientRect();
+          setCursorPosition({
+            x: e.clientX - rect.left - 100,
+            y: e.clientY - rect.top - 25,
+          });
+        }}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)